src/components/isolationMode/IsolatedAssetModal/messages.ts

```diff
@@ -14,7 +14,7 @@ export default defineMessages({
     'Isolated assets can only be borrowed up to a specific debt ceiling. Once that is reached it will not be possible to borrow more.',
   whoCanUseIt: 'Who can use it',
   whoCanUseItDescription:
-    'To deposit an asset in isolation mode you should have any other deposited positions being used as collateral and no borrow positions in your account.',
+    'To supply an asset in isolation mode you should have any other supplied positions being used as collateral and no borrow positions in your account.',
```

Comment:
```suggestion
    'To supply an asset in isolation mode you must not have any collateral enabled positions and no open borrow positions in your account.',
```

I think there should be a **not**, not sure if the rest of my suggestion makes sense.

Reply:
agree this is way better

Reply:
908686396c37e3793eadc48c54caabb44571c862